### PR TITLE
API

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description

- Integrated API
- Updated README
- Ran linters check

# Screenshots

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+const score = document.getElementById('score');
 
-updateList(scoreList, scoreBoard);
\ No newline at end of file
+async function updateScore() {
+  const fetchAPI = await getScore();
+  const updatedList = fetchAPI.result;
+  updatedList.sort((a, b) => a.score - b.score);
+  updateList(updatedList);
+}
```
Comment (julie-ify) [inline src/index.js]: - [x] Kindly refactor this method to an arrow function.

File: src/modules/scorelist.js [hunk 2]
```diff
@@ -1,24 +1,11 @@
...
-  { name: 'Dami', score: 62 },
-  { name: 'King', score: 70 },
-  { name: 'Casey', score: 18 },
-  { name: 'Greta', score: 44 },
-  { name: 'Daphine', score: 48 },
-  { name: 'Jason', score: 88 },
-];
-
-const updateList = (list, listContainer) => {
+export default function updateList(result) {
```
Comment (julie-ify) [inline src/modules/scorelist.js]: - [x] Kindly change to an arrow function

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @joshualyngdoh,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (joshualyngdoh) [reply src/index.js]: Thank you for the feedback

File: src/modules/scorelist.js (hunk 2 above)
Comment (joshualyngdoh) [reply src/modules/scorelist.js]: Thank you for the feedback

**Review by TadesseDev — APPROVED** [review]
### Hi @joshualyngdoh 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you

**Pull request merged**